utils/linkage.py

```diff
@@ -131,3 +135,41 @@ def get_street_from_address_line_1(address_line_1: str) -> str:
             string.append(key)
 
     return " ".join(string)
+
+
+def name_rank(first_name: str, last_name: str) -> list:
+    """Returns a score for the rank of a first name and last name in the US
+    https://github.com/philipperemy/name-dataset
+
+    Args:
+        first_name: any string
+        last_name: any string
+    Returns:
+        name rank for first name and last names
+        1 is the most common name, only for names in the 'United States'
+        first element is the element corresponds to the rank of the first name
```

Comment:
Let's add tests (doesn't need to be a specific number but you can check the datatype of the outputs, how they handle none, etc...)